lib/model/contact.dart

```diff
@@ -0,0 +1,54 @@
+import 'id.dart';
+import 'location.dart';
+import 'name.dart';
+import 'picture.dart';
+import 'registered.dart';
+import 'date_of_birth.dart';
+import 'login.dart';
+
+class Contact {
```

Comment:
We ussualy have Class constructor at the top, then te factory method and last the definition of variables

Reply:
done